project/tests/test_equilbration.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+import pytest
+from pymbar import testsystems
+
+from project.src.analysis.equlibration import (
+    is_equilibrated,
+    trim_non_equilibrated,
+)
+
+from .base_test import BaseTest
+
+
+class TestEquilibration(BaseTest):
+    def test_is_equilibrated(self):
+        data = testsystems.correlated_timeseries_example(
+            N=1000, tau=200, seed=432
+        )
+        assert not is_equilibrated(data, threshold=0.80)
+        assert not is_equilibrated(data, threshold=0.40)
+        assert is_equilibrated(data, threshold=0.10)
+
+    def test_incorrect_threshold(self):
+        data = testsystems.correlated_timeseries_example(
+            N=1000, tau=200, seed=432
+        )
+        with pytest.raises(ValueError, match=r"Passed \'threshold\' value"):
+            is_equilibrated(data, threshold=2.0)
+
+        with pytest.raises(ValueError, match=r"Passed \'threshold\' value"):
+            is_equilibrated(data, threshold=-2.0)
```

Comment:
is one of these supposed to be negative?

Reply:
Yeah I just wanted to make sure my validation check for `threshold` was behaving as expected. Only allow floats between 0.0-1.0